packages/performance-tests/test/GitUtils.ts

```diff
@@ -0,0 +1,12 @@
+import util = require("node:util");
+import childPoc = require("node:child_process");
```

Comment:
in iTwin.js JavaScript we prefer not to use the `node:` prefix in imports. Even if I agree it's better, it's inconsistent with our other code

Reply:
done